src/Propel/Runtime/ActiveQuery/ModelCriteria.php

```diff
@@ -2144,48 +2128,9 @@ public function doSelect(?ConnectionInterface $con = null)
             $con = Propel::getServiceContainer()->getReadConnection($this->getDbName());
         }
 
-        $this->configureSelectColumns();
-
         return parent::doSelect($con);
     }
 
-    /**
-     * @return void
-     */
-    public function configureSelectColumns()
```

Comment:
It's great that this function becomes superfluous, but since it's part of the public interface of a crucial class, it's probably better to keep it around as a no-op, with an `@deprecated` phpdoc tag.